### DISCUSSION TITLE
Prime Video not working

### DISCUSSION THREAD

**Discussion by andreaconfa** [Ysurac/openmptcprouter#2449] [Q&A]
Hi all,
i found that Amazon Prime Video is not working due tu VPN detected.
I tried to bypass all amazon services via OMR-BYPASS and also with ASN but the result is the same.

![image](https://user-images.githubusercontent.com/44619437/182456932-14f9069b-47fa-4d0e-a16d-04c749a37e4a.png)

Netflix working fine

Have anyone found a way to bypass this issue? 
Thanks

**Comment by Ysurac** [comment]
Selecting "AmazonVideo" in "protocols and services" work, but new DNS requests must be done. If the client have some DNS cache this can take time.

**Reply by andreaconfa** [reply]
> Selecting "AmazonVideo" in "protocols and services" work, but new DNS requests must be done. If the client have some DNS cache this can take time.

Even after a dns flush "ipconfig -flushdns" still not working

**Reply by Ysurac** [reply]
I also had some strange problems, maybe Amazon keep the IP in a cache before saying it's safe without VPN.
(I'm using Amazon Video every day over OpenMPTCProuter)

**Comment by Chris5342** [comment]
Is this for a TV?

I find it works best to bypass the TV with MAC eg:

![Screenshot 2022-08-07 at 08 17 03](https://user-images.githubusercontent.com/105393840/183279922-20725fa0-fce8-4952-9566-7034b0851736.jpg)

If you don't want to bypass the TV you could bypass a dongle device eg firetv, appletv or googletv and use that for prime video.

I have a TV and appletv and googletv all with MAC bypass.

It's useful to have multiple devices anyway as sometimes streaming doesn't work one device for another.
